docs/api/specs/browserResidentDesktopAgents.md

```diff
@@ -0,0 +1,253 @@
+
+:::
+
+:::tip
+
+When referencing "DA" in this document we will hereafter always mean a "Browser-Resident Desktop Agent" - code that runs in a browser page (iframe or window) and which conforms to this specification.
+
+:::
+
+## Launching apps
+
+As a prerequisite for running FDC3 in the browser, a DA must first exist as running code in a browser window (See failover functions for an exception to this rule), although that code MAY also connect to or rely on remotely hosted services. We will refer to this window as the "DA Window".
+
+As the DA typically acts as a launcher for applications, it will often be the case that the DA window is related to the application window(s) in that it may have create the application window with `window.open()` or by creating an iframe and loading the application URL into it. Hence, the DA window may be referred to as a 'parent' (window or frame) of the application frame and the relationship may be used to implement communication between the frames.
+
+:::note
+
+It is possible to have multiple DA Windows. For instance, a DA may propagate itself into new windows. Communication and coordination between DA Windows is an implementation detail and is not covered by this specification.
+
+:::
+
+When an app runs `getAgent()`, it checks for the existence of `window.parent`, `window.opener` and `window.parent.opener` (and will continue up the chain of parent frames, e.g. `window.parent.parent`, `window.parent.parent.opener` until the reference to the next parent is equal to the current one (e.g. `window.parent.parent === window.parent` indicating that the frame does not have a parent). `getAgent()` will then send a standardized `WCP1Hello` message to each parent window or frame reference via [`postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) in to discover a DA.
+
+Hence, apps may be launched:
+
+1. By creating iframes in a DA Window
+2. By calling `window.open` from a DA Window
+3. By creating iframes in a window that was opened from a DA Window
+
+and the Desktop Agent application will be found in a 'parent' of the application frame.
+
+## Responding to app instance connections - Web Connection Protocol (WCP)
+
+Browser Resident DAs MUST call `window.addEventListener("message",...)` to receive incoming connection requests from apps, in the form of [`"WCP1Hello"`](https://fdc3.finos.org/schemas/next/api/WCP1Hello.schema.json) messages defined in the [Web Connection Protocol](./webConnectionProtocol).
```

Comment:
Missing space after comma in window.addEventListener

Reply:
Fixed in e3ce1dbc8c18624e42dcb5090059cf93daae20e2